packages/core/jest.config.js

```diff
@@ -17,4 +18,5 @@ module.exports = {
         '^Services$': '<rootDir>/src/Services/index.js',
         '^Stores/(.*)$': '<rootDir>/src/Stores/$1',
     },
+    transformIgnorePatterns: ['/node_modules/(?!(@deriv-com/quill-ui|@simplewebauthn/browser)).+\\.js$'],
```

Comment:
Without changes in this file we can't use quill components in tests (only if we mock them)

Reply:
can we add @deriv-com/quill-ui in the jest.config.base.js so that it's applied for all the packages?